SConstruct

```diff
@@ -2185,10 +2185,11 @@ if any(target.startswith('test') for target in COMMAND_LINE_TARGETS):
 
     if env['python_package'] == 'none':
         # copy scripts from the full Cython module
+        # (skipping 'yaml2ck', which depends on the full Python module)
         test_py_int = env.Command('#build/python_local/cantera/__init__.py',
                                   '#interfaces/python_minimal/cantera/__init__.py',
                                   Copy('$TARGET', '$SOURCE'))
-        for script in ["ck2yaml", "ctml2yaml", "yaml2ck"]:
+        for script in ["ck2yaml", "ctml2yaml", "cti2yaml"]:
```

Comment:
I believe `cti2yaml` also requires the full module, to be able to read the CTI file?

Reply:
Afaik, it needs the full package just for validation? (same approach as `ck2yaml`) …

Reply:
Yep, that's right, I thought the CTI itself was read by Cantera to parse it, but it's just `exec`ed by `cti2yaml` to turn it into Python objects.